flake.lock

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Is it necessary to commit this file as well for the Flake to be installed directly? If possible I would like to avoid committing lock-files.